feat(armory-io): added documentation for the expose preview links via webhooks (#1919)
armory/docs: content/en/includes/cdaas/deploy/preview-link-details.md

You can configure multiple `exposeServices` entries, each having a different expiration time.

> The links to the exposed services would be available for further use in webhooks. 
> The naming convention for such links is following `armory.preview.<service-name-1>`. 

Reviewer: ```suggestion
The links to the exposed services will be available for further use in webhooks. 
The naming convention for such links follows: `armory.preview.<service-name-1>`. 
```
